charts/kubernetes-agent/values.yaml

```diff
@@ -191,6 +191,13 @@ scriptPods:
                 - arm64
                 - amd64
 
+  # -- The repository, pullPolicy & tag to use for the script pod image. If left blank, will use the `octopusdeploy/kubernetes-agent-tools-base` image.
+  # @section -- Script pod values
+  image:
```

Comment:
Will this be a pain since the `image` is in the `deploymentTarget` section in the v2 of the chart?

Reply:
yes, however I didn't want to backport some of the values structure changes. The migration code in V2 should handle this.

Reply:
The forward merge will be a bit fiddly, but should be manageable